README.md

```diff
@@ -142,6 +142,9 @@ if (templateValues.eslint !== 'y') {
 ```
 Note that the string does contain the .ejs suffix, since the ignore applies when reading the file, the string should exactly match the name of the file in your template.
 
+##### `ignoredFolders`
+When the generator goes through your template, it will ignore any folders provided in this array.
```

Comment:
do you have to mention the entire path?
```
template
|-src
  |-components
    |-Welcome
      |-Welcome.jsx
      |-Welcome.css
```
     
Would I specify 'Welcome', 'src/components/Welcome', or some thing else?

Either way, could this piece of readme give an example such as the above to make it ultra clear?

Reply:
You would just have to specify the name of the folder ( case sensitive ). I can add an example here.
Here is how it would be done for the above
```
ignoredFolders: ['Welcome']
```